src/child_process.ml

```diff
@@ -28,6 +28,8 @@ type execFileOptions =
 external execFile :
   string -> string array -> execFileOptions -> 'a callback -> unit = "" [@@bs.module "child_process"]
 
-let execFile cmd args options cb =
+let execFile cmd args ?cwd ?env ?encoding ?timeout ?maxBuffer ?killSignal ?uid ?gid ?windowsHide ?windowsVerbatimOptions ?shell cb =
+  let options = execFileOptions ?cwd:cwd ?env:env ?encoding:encoding ?timeout:timeout ?maxBuffer:maxBuffer ?killSignal:killSignal ?uid: uid ?gid:gid ?windowsHide: windowsHide ?windowsVerbatimOptions:windowsVerbatimOptions ?shell:shell ()
```

Comment:
You can leave out the variable value when it has the same name as the named argument:
```
let foo = "bla" in
f ~foo
```

Reply:
I would usually put the optional arguments up front before anonymous ones but it's a matter of personal taste..